src/sentry/utils/cloudfunctions.py

```diff
@@ -115,7 +115,7 @@ def delete_function(funcId):
     client = CloudFunctionsServiceClient()
     client.delete_function(
         request=DeleteFunctionRequest(
-            name=f"projects/{SENTRY_FUNCTIONS_PROJECT_NAME}/locations/{SENTRY_FUNCTIONS_REGION}/functions/fn-{funcId}",
+            name=f"projects/{settings.SENTRY_FUNCTIONS_PROJECT_NAME}/locations/{settings.SENTRY_FUNCTIONS_REGION}/functions/fn-{funcId}",
```

Comment:
@vpandiarajan20 seems like a lot of duplicated code here, is it possible to clean up?